packages/amplify-graphql-model-transformer/src/definitions.ts

```diff
@@ -5,6 +5,12 @@ export const FLOAT_CONDITIONS = ['ne', 'eq', 'le', 'lt', 'ge', 'gt', 'between'];
 export const BOOLEAN_CONDITIONS = ['ne', 'eq'];
 export const SIZE_CONDITIONS = ['ne', 'eq', 'le', 'lt', 'ge', 'gt', 'between'];
 
+export const SUBSCRIPTION_STRING_CONDITIONS = ['ne', 'eq', 'le', 'lt', 'ge', 'gt', 'contains', 'notContains', 'between', 'beginsWith', 'in', 'notIn'];
+export const SUBSCRIPTION_ID_CONDITIONS = ['ne', 'eq', 'le', 'lt', 'ge', 'gt', 'contains', 'notContains', 'between', 'beginsWith', 'in', 'notIn'];
+export const SUBSCRIPTION_INT_CONDITIONS = ['ne', 'eq', 'le', 'lt', 'ge', 'gt', 'between', 'in', 'notIn'];
+export const SUBSCRIPTION_FLOAT_CONDITIONS = ['ne', 'eq', 'le', 'lt', 'ge', 'gt', 'between', 'in', 'notIn'];
+export const SUBSCRIPTION_BOOLEAN_CONDITIONS = ['ne', 'eq'];
```

Comment:
nit: could re-use `BOOLEAN_CONDITIONS` since it's always equal or not equal.

Reply:
Since the values change for the other scalar types, I had to create another set of constants for subscriptions. I've created new type for boolean to keep the scalar declarations uniform.